dmscripts/bulk_upload_documents.py

```diff
@@ -0,0 +1,39 @@
+import os
+import re
+import sys
+sys.path.insert(0, '.')
+from dmutils.documents import get_countersigned_agreement_document_path
+
+
+def upload_file(bucket, dry_run, file_path, framework_slug):
+    supplier_id = get_supplier_id_from_framework_file_path(file_path)
+    upload_path = get_countersigned_agreement_document_path(framework_slug, supplier_id)
+    if not dry_run:
+        with open(file_path) as file_contents:
+            bucket.save(upload_path, file_contents, acl='public-read', move_prefix=None)
+        print(supplier_id)
```

Comment:
We should probably be using the [logging infrastructure](https://github.com/alphagov/digitalmarketplace-scripts/blob/e03a8c15276440e44a5c2104034f35523827e177/scripts/index-services.py#L28) for these print statements now that we have it.